src/components/header/Header.js

```diff
@@ -0,0 +1,16 @@
+import Navbar from './navigationBar';
```

Comment:
also create styles.css file in this folder and move all styles related to header to this file. And add `import './styles.css'` here